pkg/apis/apps/v1/defaults_test.go

```diff
@@ -175,8 +175,19 @@ func TestSetDefaultDaemonSetSpec(t *testing.T) {
 	}
 }
 
+func getMaxUnavailable(maxUnavailable int) *intstr.IntOrString {
+	maxUnavailableIntOrStr := intstr.FromInt(maxUnavailable)
+	return &maxUnavailableIntOrStr
+}
+
+func getPartition(partition int32) *int32 {
+	return &partition
+}
+
 func TestSetDefaultStatefulSet(t *testing.T) {
 	defaultLabels := map[string]string{"foo": "bar"}
+	defaultMaxUnavailable := intstr.FromInt(1)
+	notTheDefaultMaxUnavailable := intstr.FromInt(3)
```

Comment:
delete these variables now that you have the functions (there are some references too).